### PR TITLE
Grant more folks owners karma.

### PR DESCRIPTION
**Pull request by rukletsov**
## High-level description

This feature improves productivity and velocity.

## Checklist for all PRs

  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by rukletsov** [comment]
@mesosphere-mergebot bump-ee

**Comment by rukletsov** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm https://jira.mesosphere.com/browse/DCOS-41676

File: owners.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "bbannier": {"slack": "benjamin", "jira": "benjamin.bannier"},
```
Comment (bbannier) [inline owners.json]: I am not sure why we use handles for the Mesosphere-internal slack here, but this seems consistent with the rest of the file.

For posterity, my DC/OS slack handle would be `bbannier` and Till's is `till.mesosphere`.

**Comment by rukletsov** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/hybrid-cloud https://jira.mesosphere.com/browse/DCOS-41673

**Comment by gpaul** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm https://jira.mesosphere.com/browse/DCOS-41676

**Comment by gpaul** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/hybrid-cloud https://jira.mesosphere.com/browse/DCOS-41673

**Comment by rukletsov** [comment]
@mesosphere-mergebot label Ready For Review

**Pull request merged**